src/prisma/schema.prisma

```diff
@@ -1403,6 +1406,7 @@ model MemberProperty {
   permissions          MemberPropertyPermission[]
   enabledViews         VisibilityView[]           @default([gallery, table, profile])
   required             Boolean                    @default(false)
+  primaryIdentity      Boolean                    @default(false)
```

Comment:
there's a danger adding this as an option to all properties that multiple ones could be flagged as primary. it seems like instead this should live on the space table?